internal/database/database.go

```diff
@@ -49,7 +49,7 @@ func NewDB(cfg *domain.Config, log logger.Logger) (*DB, error) {
 		if cfg.PostgresHost == "" || cfg.PostgresPort == 0 || cfg.PostgresDatabase == "" {
 			return nil, errors.New("postgres: bad variables")
 		}
-		db.DSN = fmt.Sprintf("postgres://%v:%v@%v:%d/%v?sslmode=disable", cfg.PostgresUser, cfg.PostgresPass, cfg.PostgresHost, cfg.PostgresPort, cfg.PostgresDatabase)
+		db.DSN = fmt.Sprintf("postgres://%v:%v@%v:%d/%v?sslmode=%v", cfg.PostgresUser, cfg.PostgresPass, cfg.PostgresHost, cfg.PostgresPort, cfg.PostgresDatabase, cfg.PostgresSSLMode)
```

Comment:
We could improve it further by allowing extra args in case anyone wants that. New config variable called `PostgresExtraParams` maybe, and append it to the string if not empty.

```diff
		db.DSN = fmt.Sprintf("postgres://%v:%v@%v:%d/%v?sslmode=%v", cfg.PostgresUser, cfg.PostgresPass, cfg.PostgresHost, cfg.PostgresPort, cfg.PostgresDatabase, cfg.PostgresSSLMode)
+               if cfg.PostgresExtraParams != "" {
+                   db.DSN = fmt.Sprintf("%s&%s", db.DSN, cfg.PostgresExtraParams)
+               }
```

Reply:
Hi! Indeed, everyone should be able to add other params handled by the pq lib. I've applied your suggestion